Backends/CLX/fonts.lisp

```diff
@@ -41,7 +41,7 @@
     (when fonts
       (xlib:open-font display (first fonts)))))
 
-(defmethod text-style-mapping ((port clx-port) text-style &optional character-set
+(defmethod text-style-mapping ((port clx-port) (text-style text-style) &optional character-set
                                &aux (text-style (climb:parse-text-style* text-style)))
```

Comment:
Isn't `parse-text-style*` the identity when `text-style` already is a `text-style` instance?

Reply:
no, text-style may be incomplete (i.e it may have NIL components). It would be the identity for `parse-text-style` (without a star)

Reply:
Right, I already forgot. But that reminds me: can we rename `parse-text-style*` to `ensure-fully-specified-text-style` or similar?

Reply:
I'm not much into `overly-long-and-descriptive-name`'s, also it is indeed a parse-text-style variant, so "starred" name makes perfect sense here to me. note, that it is not only to ensure that the text style is fully specified but also to normalize the size

Reply:
Fair enough. I might bring it up again later, though :)